tests/manage/test_disruptive.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import logging
+import inspect
+from ocs import constants
+from tests import helpers, disruption_helpers
+from ocsci.testlib import ManageTest, tier4
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret, create_fs_secret, create_ceph_filesystem
+)
+
+logger = logging.getLogger(__name__)
+
+DISRUPTION_OPS = disruption_helpers.Disruptions()
+
+
+def get_operation_obj(operations_instance, operation):
+    """
+    Get the instance method object along with the method name according the corresponding needed function name
+
+    Args:
+        operations_instance (Class instance): The instance of the class
+        operation (str): The name of the operation to look for
+
+    Returns:
+        tuple: instance method object along with the method name
+    """
+    operations_pool = [i for i in inspect.getmembers(operations_instance, predicate=inspect.ismethod)]
+    ret = [i for i in operations_pool if i[0] == operation][0]
+    return ret
+
+
+class BaseDisruption(ManageTest):
+    """
+    Base class for PVC related disruption tests
+    """
+
+    def disruptive_base(self, operation_to_disrupt, disruption):
+        """
+        Base function for PVC disruptive tests
+        """
+        dis_op = get_operation_obj(operations_instance=DISRUPTION_OPS, operation=disruption)
+
+        pvc_obj = helpers.create_pvc(
+            sc_name=self.sc_obj.name, wait=False
+        )
+        if operation_to_disrupt == 'create_pvc':
+            dis_op[1]()
```

Comment:
This is taking too long to invoke and therefore no effect on the pvc creation

Reply:
see new commit. 
Now I'm getting the resource to harm, then creating the resource (pvc/pod) and then deleting the resource.
Also, the resource creation doesn't include ocs.reload() so it allows an immediate disruption, right after the resource creation